server/image_status_libpod.go

```diff
@@ -0,0 +1,67 @@
+// +build libpod
+
+package server
+
+import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+// ImageStatus returns the status of the image.
+func (s *Server) ImageStatus(ctx context.Context, req *pb.ImageStatusRequest) (resp *pb.ImageStatusResponse, err error) {
+	const operation = "image_status"
+	defer func() {
+		recordOperation(operation, time.Now())
+		recordError(operation, err)
+	}()
+
+	logrus.Debugf("ImageStatusRequest: %+v", req)
+
+	image, err := s.ImageRuntime().NewFromLocal(req.Image.Image)
+	if err != nil {
+		return nil, err
+	}
+
+	// Discarding the err here as I dont believe not getting size should be fatal
+	size, _ := image.Size(getContext())
```

Comment:
We have a `ctx` argument here too, so we can use that instead of `getContext()` ([previous comment][1]).  That allows us to drop `getContext` entirely.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1669#discussion_r199626976

Reply:
fixed